### ISSUE TITLE
Redo forecast catch levels 

### ISSUE THREAD

**Issue by allanhicks** [cgrandin/hake-assessment#155]
We will need to recalculate the forecast catch levels for the last three rows of the table. if anyone would like to help with that to expeience how I do it, let me know. It could be automated using a simple search algorithm, which I think I coded once a few years ago. But, it will take about 10 minutes to figure out.

**Comment by aaronmberger** [comment]
It would be good for me.  You want to do a hangout on Monday?

On Saturday, February 27, 2016, allanhicks <notifications@github.com> wrote:

> We will need to recalculate the forecast catch levels for the last three
> rows of the table. if anyone would like to help with that to expeience how
> I do it, let me know. It could be automated using a simple search
> algorithm, which I think I coded once a few years ago. But, it will take
> about 10 minutes to figure out.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cgrandin/hake-assessment/issues/155>.
>


-- 

*Aaron M. Berger, Ph.D. Research Mathematical Statistician*
*Fishery Resource Analysis and Monitoring Division*
*Northwest Fisheries Science Center*
*2032 SE OSU Dr. Bldg. 955*
*Newport, OR 97365-5275*



*aaron.berger@noaa.gov <aaron.berger@noaa.gov>541.867.0562*

**Comment by cgrandin** [comment]
I'd like to see this done! If it's too much to organize at this late stage, then maybe Aaron could show us the method next JTC meeting or something.

**Comment by taylori** [comment]
When we all split with ferries to catch at whatnot, we didn't have a chance
to discuss timelines for finishing the document. It seems that we have the
potential to get it done in record time this year, but also don't need to
kill ourselves.

Once the document is finished, if folks aren't too swamped with other
things, or burned out on hake, I'd be interested in having a follow-up
phone meeting where we could walk through some of the code that would be
good for more of us to understand, especially things that Allan has been in
charge of. Allan and I will be able to meet more frequently to do some
hand-off of code but a group discussion might be a good idea too.

**Comment by cgrandin** [comment]
That sounds great, Ian. Once the document is done, I won't betoo swamped so I'll have some time to fix up some aspects of the code and to hopefully learn how this works. Maybe we could incorporate code into the project to find these numbers automatically, as Allan mentioned he has already done.

**Comment by allanhicks** [comment]
If you want to join a Google Hangout this morning, I'm free.  Let me know
what time works best, and I'll set up a hangout. Probably best to do this
soon so we can source all.r again for the proper forecasts.  If you don't
have time this morning, we can go over this later.

**Comment by andrew-edwards** [comment]
I'm around so just let us know. Chris is in work also. Set up what works for you and we can try and make it.

**Comment by cgrandin** [comment]
I think we were successful here (thanks Allan and Aaron), so closing.
